apps/pie-microsite/src/content/pages/foundations/iconography/overview.md

```diff
@@ -0,0 +1,184 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: Iconography
+    order: 1
+---
+
+## Icon sets
+
+We have two icon sets that can be used in our products and communications.
+
+### Small icon set
+
+This is the default icon set used throughout our products across Web, Android and iOS. It features less details, which helps with making them simpler when used at smaller sizes.
+
+{% contentPageImage {
+    src:"../../../../../assets/img/foundations/iconography/small-icon-set.svg",
+    width: "33.91",
+    caption: "The example above shows what our ‘Kebab bowl’ icon looks like on the Small set."
+} %}
+
+### Large icon set
+
+This set should be used for illustrative purposes, such as communications or marketing materials. The icons are more detailed, which helps make them look better when used at bigger sizes.
+
+{% contentPageImage {
+    src:"../../../../../assets/img/foundations/iconography/large-icon-set.svg",
+    width: "65.65px",
+    caption: "The example above shows what our ‘Kebab bowl’ icon looks like on the Large set."
+} %}
+
+---
+
+## Icon anatomy
+
+Our icons are always placed inside a bounding box, which ensures that vertical and horizontal spacing always adhere to the guidance shown in our Spacing guides. When exporting icons for your products, please export the whole Bounding Box, instead of just the Icon Area.
+
+{% contentPageImage {
+    src:"../../../../../assets/img/foundations/iconography/icon-anatomy.svg",
+    width: "199px"
+} %}
+
+{% list {
+    type: "ordered",
+    items: [
+        "**Icon area:** (Small (Default size): 14x14px / Large (Default size): 28x28px).",
+        "**Bounding box:** (Small (Default size): 16x16px / Large (Default size): 32x32px)."
```

Comment:
just checking - do we guarantee that the inputted item order is preserved?

Reply:
Yep, as discussed, we have a snapshot test for it